cfgov/unprocessed/js/modules/youtube-api.js

```diff
@@ -0,0 +1,86 @@
+/* ==========================================================================
+   YouTube API
+   ========================================================================== */
+
+import jsLoader from './util/js-loader';
+
+const IMAGE_URL = 'https://img.youtube.com/vi/%video_id%/maxresdefault.jpg';
+const SCRIPT_API = 'https://www.youtube.com/iframe_api';
+
+/**
+ * The "onYouTubeIframeAPIReady" function is automatically called when the
+ * YouTube iFrame API is done loading.
+ * See https://developers.google.com/youtube/iframe_api_reference
+ * @param {Function} callback - function to call when the API is ready.
+ */
+function attachAPIReadyCallback( callback ) {
+  window.onYouTubeIframeAPIReady = callback;
+}
+
+/**
+ * Embed the YouTube iFrame API.
+ * @param {Function} [callback] - function to call when the script is loaded.
```

Comment:
```suggestion
 * @param {Function} callback - function to call when the script is loaded.
```

Reply:
This is correct with the brackets. Those denote [optional arguments in jsdocs](https://devhints.io/jsdoc). Where this is called, [here](https://github.com/cfpb/cfgov-refresh/pull/5162/files#diff-9efea6479fb7879a333f4d6718297e33R71), the callback is not supplied. We could remove it entirely, but it is useful for debugging and has been carried over from the prior codebase.

Reply:
Ah, whoops! That makes sense.